examples/gatsby/src/pages/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Media } from "../Media"
+
+export default function IndexPage() {
+  return (
+    <>
+      <Media at="xs">Hello mobile view</Media>
```

Comment:
```suggestion
      <Media at="xs">Hello mobile view!</Media>
```

#trivial but consistent with your level of excitement in the next line.